### ISSUE TITLE
Miller columns at larger screen sizes

### ISSUE THREAD

**Issue by 12people** [tchx84/Portfolio#295]
I would love a file browser on Linux that has a miller column view, as that view makes it much easier to navigate complex folder hierarchies.

It seems like a fitting expansion of Portfolio for larger devices (e.g. tablets, 2-in-1s), as Portfolio is already based around a single-column list of files. And it would also bring a [much-requested Nautilus feature](https://gitlab.gnome.org/GNOME/nautilus/-/issues/27) to Flathub.

Unfortunately, Nautilus has deemed this feature as out of scope, but since Portfolio has a much more recent architecture and a lot less old code to maintain, I'm hoping it would be easier to implement a Miller Column view here.

**Comment by kitenium** [comment]
This app looks promising. Does it support linux desktop?

**Comment by 12people** [comment]
> This app looks promising. Does it support linux desktop?

Yes, you can get it on [Flathub](https://flathub.org/apps/details/dev.tchx84.Portfolio). However, it's touch-centric and doesn't support Miller columns yet.

**Comment by PHHENS** [comment]
There is a great file browser called katana here :
[https://github.com/gavr123456789/Katana](https://github.com/toolstack/Folio/issues/113)
(Take a look at the videos)
Following this design (miller columns not proposed by nautilus)
could make Portfolio usefull for mobile and laptop platform.
